_docs/install-and-upgrade.md

```diff
@@ -202,6 +202,7 @@ supported (even when hosted on this website), so use them at your own risk:
 - [Ansible role using Supervisor](https://github.com/astorije/ansible-thelounge)
 - [ARMHF Docker images](https://hub.docker.com/r/lsioarmhf/thelounge/)
 - [YunoHost app](https://github.com/YunoHost-Apps/thelounge_ynh)
+- [Cloudron app](https://cloudron.io/store/io.github.thelounge.html) [Source](https://git.cloudron.io/cloudron/thelounge-app)
```

Comment:
Can you remove Source? Looks odd in text.

Reply:
@xPaw will do